plugins/ci/skills/fetch-prow-job-runs/fetch_prow_job_runs.py

```
@@ -0,0 +1,133 @@
+        "sort": "desc",
+    }
+    url = "%s?%s" % (BASE_URL, urllib.parse.urlencode(params))
+    try:
+        with urllib.request.urlopen(url, timeout=60) as resp:
+            body = resp.read().decode("utf-8")
+    except urllib.error.HTTPError as e:
+        detail = ""
+        try:
+            detail = e.read().decode("utf-8")
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)